pkg/apiserver/user/sqlauth/sqlauth.go

```diff
@@ -9,7 +13,36 @@ import (
 	"github.com/pingcap/tidb-dashboard/pkg/tidb"
 )
 
-const typeID utils.AuthType = 0
+const (
+	typeID utils.AuthType = 0
+
+	// Base privileges
+	privAll             string = "ALL PRIVILEGES"
+	privProcess         string = "PROCESS"
+	privShowDB          string = "SHOW DATABASES"
+	privSystemVarAdmin  string = "SYSTEM_VARIABLES_ADMIN"
+	privSuper           string = "SUPER"
+	privConfig          string = "CONFIG"
+	privDashboardClient string = "DASHBOARD_CLIENT"
+	// Extra privileges when TiDB SEM is enabled
+	privRestrictedTablesAdmin string = "RESTRICTED_TABLES_ADMIN"
+	privRestrictedStatusAdmin string = "RESTRICTED_STATUS_ADMIN"
```

Comment:
Seems that these constants cannot be reused in other places. May be better to be not a constant.

Reply:
OK